initial commit

Anton-Kuchmasov opened the pull request:
[DEMO LINK](https://Anton-Kuchmasov.github.io/layout_landing-page/)

ArtemTeslenko reviewed (changes requested):
Great 👍 There are some points
1. In header should be a phone number when hover [screen](https://prnt.sc/VGQDn2-WYLMK), also it should offer to call
2. Call to order should not redirect us to other page [screen](https://prnt.sc/9-kesycbOOzD)
3. Font-size should be 56px on desktop [screen](https://prnt.sc/exaEhLbbuQPL)
4. It seems that transition works in one direction, transition should be on element, not on hover [screen](https://prnt.sc/Q42r5v7dL9ls)
5. Font-size 18px on desktop [screen](https://prnt.sc/FYK8-SztxqTy)
6. Image has extra space under, try `display: block` it will help [screen](https://prnt.sc/qa27e7c1zpaQ), please check all over the document
7. Very bad practice with margin [screen](https://prnt.sc/RK3HcUqSGwPv), [screen](https://prnt.sc/QlorUa0avDBY). 1 - element should have top OR bottom margin in such case; 2 - there you have margin collapse 😥
8. There you have margin which push the parent element, not good [screen](https://prnt.sc/uiHviSMXSGCM). If you want one section push another section you can use margin, if you need space inside element, you should use padding. Try to use margin between siblings. Also here should be 128px between sections you have 60 + 96 [screen](https://prnt.sc/7vzNN5aWJmXp)
9. Image on image when hover [screen](https://prnt.sc/88tMBjeF-_DD)
10. Margin between child and parent again, the same with button [screen](https://prnt.sc/DbEprg_ZyIlD), [screen](https://prnt.sc/M_2JUAZsafRp)
11. And here you were bitten by your margins 60 + 56 + 120, when it should be just 120px [screen](https://prnt.sc/MEMlB_McOV-M)
12. In inputs should be Space Mono [screen](https://prnt.sc/b1bpMN50-o5R)
13. [Checklist](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md) p. 15
14. Forbid to change textarea size [screen](https://prnt.sc/-SE0BsrevTsm)
15. Too much space between elements [screen](https://prnt.sc/rBlGoZtoeRvv)
16. Add some hover for phone, email and address [screen](https://prnt.sc/mh-T09Ctf_Vu)

Anton-Kuchmasov commented:
Thanks for really huuuuge review report, it can be so useful for me as student! 

1) fixed (added wrap to cell-icon + p>number), now it works great; 
2) changed only `href="tel:+134124...."`, now it works well;
3) just forgot to added onDesktop mixin :( fixed;
4) fixed an issue with hover effect;
5) done without descripton. media query -> fz:18;
6)` display: block` really helped, thx;
7) fixed with p:0 m:0 on h3-class selector;
8) margin was fixed, p:0;
9) fixed an scale() value to 1.04, it's allright now;
10) commited changes without margin between p and c, now it's looking really better;
11) deleted an unnessesary margin in .categories;
12) now it's Space Mono, sure;
13) added `<form onsubmit="this.reset(); return false;">` up to form implementation;
14) fixed textarea size;
15) fixed it to `gap: 14px`;
16) added class "item-link" to making bold on hover.

roman-mirzoian commented on src/utils/variables.scss:
```diff
@@ -0,0 +1,9 @@
+$tablet-min-width: 576px;
+$desktop-min-width: 1024px;
+
+$effectDuration: 0.25s;
+$menuScrollingDuration: 0.45s;
```
Use one style of writing variables.
```suggestion
$tablet-min-width: 576px;
$desktop-min-width: 1024px;

$effect-duration: 0.25s;
$menu-scrolling-duration: 0.45s;
```
